subprojects/testing-base/src/main/java/org/gradle/api/internal/tasks/testing/DefaultTestDescriptor.java

```diff
@@ -63,4 +63,9 @@ public String getDisplayName() {
     public String getClassDisplayName() {
         return classDisplayName;
     }
+
+    @Override
+    public boolean isParameterizedTest() {
```

Comment:
I'd move this up to `AbstractTestDescriptor` and then just override it in `ParameterizedTestSuiteDescriptor`

Reply:
😅 I should take a carefully look at the types hierarchy before overriding the method .., updated.